forgerock-integration-tests/build.gradle.kts

```diff
@@ -43,11 +43,17 @@ dependencies {
     androidTestImplementation(libs.androidx.biometric.ktx)
     androidTestImplementation(libs.nimbus.jose.jwt)
     androidTestImplementation(libs.okhttp)
+    androidTestImplementation(libs.kotlinx.coroutines.test)
+    testImplementation(libs.kotlinx.coroutines.test)
```

Comment:
do we need this testImplementation(libs.kotlinx.coroutines.test) ?,what if you remove this and run the test

Reply:
Yeah, looks like this is not needed indeed - I removed it.